test/objects.js

```diff
@@ -681,6 +681,67 @@
     assert.ok(_.isBoolean(false), 'and so is false');
   });
 
+  QUnit.test('isMap', function(assert) {
+    var keyString = 'a string', keyObj = {}, keyFunc = function() {};
+    var obj = new Map([[keyString, 'value one'], [keyObj, 'value two'],
```

Comment:
You need to ensure `Map`, `Set`, `WeakMap` and `WeakSet` exist. The tests will fail in browsers like IE10 and environments like PhantomJS or Node 0.10.0